ui/lib/core/addon/components/policy-example.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+/**
+ * @module PolicyExample
+ * PolicyExample component is meant to show an example of a policy. The PolicyExample can rendered by the PolicyTemplate
+ * component (as in the first example below). PolicyExample can also be rendered in a modal (second example below), which
+ * is how it is conditionally rendered by the PolicyForm component.
+ *
+ * @example
+ *  <PolicyExample
+ *    @policyType={{@model.policyType}}
+ *  />
+ *
+ * @example (in modal)
+ *  <Modal
+ *    @onClose={{fn (mut this.showTemplateModal) false}}
+ *    @isActive={{this.showTemplateModal}}
+ *  >
+ *    <section class="modal-card-body">
+ *      {{! code-mirror modifier does not render value initially until focus event fires }}
+ *      {{! wait until the Modal is rendered and then show the PolicyExample (contains JsonEditor) }}
+ *      {{#if this.showTemplateModal}}
+ *        <PolicyExample @policyType={{@model.policyType}}/>
+ *      {{/if}}
+ *    </section>
+ *    <div class="modal-card-head has-border-top-light">
+ *      <button type="button" class="button" {{on "click" (fn (mut this.showTemplateModal) false)}} data-test-close-modal>
+ *        Close
+ *      </button>
+ *    </div>
+ *  </Modal>
+ * ```
+ * @param {string} policyType - policy type to decide which template to render; can either be "acl" or "rgp"
+ */
+
+export default class PolicyExampleComponent extends Component {
+  // formatting here is purposeful so that whitespace renders correctly in JsonEditor
+  policyTemplates = {
```

Comment:
there's one more policy type we need to add here [EGP](https://developer.hashicorp.com/vault/tutorials/policies/sentinel#endpoint-governing-policies-egps)